src/components/Table/styles.scss

```diff
@@ -1,24 +1,133 @@
+/* stylelint-disable no-descending-specificity */
 @import "../../styles/colors";
 @import "../../styles/font-sizes";
+@import "../../styles/paddings";
+
```

Comment:
this css file will become too huge, so we need to put the head specific styles in another css file inside the head folder, and the same for body.